superbench/benchmarks/model_benchmarks/pytorch_cnn.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module of the Pytorch CNN models."""
+
+import time
+
+import torch
+from torchvision import models
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, Precision
+from superbench.benchmarks.model_benchmarks.model_base import Optimizer
+from superbench.benchmarks.model_benchmarks.pytorch_base import PytorchBase
+from superbench.benchmarks.model_benchmarks.random_dataset import TorchRandomDataset
+
+
+class PytorchCNN(PytorchBase):
+    """The CNN benchmark class."""
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+        self._supported_precision = [Precision.FLOAT32, Precision.FLOAT16]
+        self._optimizer_type = Optimizer.SGD
+        self._loss_fn = torch.nn.CrossEntropyLoss()
+
+    def add_parser_arguments(self):
+        """Add the CNN-specified arguments."""
+        super().add_parser_arguments()
+
+        self._parser.add_argument('--model_type', type=str, required=True, help='The cnn benchmark to run.')
+        self._parser.add_argument('--image_size', type=int, default=224, required=False, help='Image size.')
+        self._parser.add_argument('--num_classes', type=int, default=100, required=False, help='Num of class.')
```

Comment:
why 100 is the default num_classes instead of 1000?

Reply:
100 or 1000 has no impact for the performance.  We use 100 for all models. If you think 1000 is more suitable, we can change to 1000.

Reply:
since most of these models are originally trained on imagenet, I think 1000 is more reasonable.

Reply:
Sure, changed to 1000.